src/Scripting/CSharp/Hosting/CSharpReplServiceProvider.cs

```diff
@@ -18,7 +18,7 @@ public CSharpReplServiceProvider()
         public override DiagnosticFormatter DiagnosticFormatter => CSharpDiagnosticFormatter.Instance;
 
         public override string Logo
-            => string.Format(CSharpScriptingResources.LogoLine1, GetType().GetTypeInfo().Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>().Version);
+            => string.Format(CSharpScriptingResources.LogoLine1, GetType().GetTypeInfo().Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion.Split('+')[0]);
```

Comment:
>Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion.Split('+')[0] [](start = 89, length = 103)

Could this be provided as a helper method from Microsoft.CodeAnalysis? The method could be internal since the Scripting assemblies have IVT access.